frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs

```diff
@@ -358,7 +182,14 @@ createAccountConfig ideL renderErrors splashWF name chainId keyset = Workflow $
       Nothing
       $ uiAccountDropdown def (pure $ \_ a -> fromMaybe False (accountHasFunds a)) (pure id)
 
-  let payload = HM.singleton tempkeyset $ toJSON keyset
+  let payload = HM.singleton tempkeyset $ toJSON' keyset
```

Comment:
I think this whole function is unused now.

Reply:
good catch